src/spoke/interfaces/ISpoke.sol

```diff
@@ -475,6 +475,14 @@ interface ISpoke is ISpokeBase, IMulticall, INoncesKeyed, IAccessManaged {
     address user
   ) external view returns (UserPosition memory);
 
+  /// @notice Returns the most up-to-date user account data information.
+  /// @dev Utilizes user's current dynamic configuration of user position.
+  /// @dev Risk premium is calculated from up-to-date user position data, and will likely differ from the stored user risk premium.
+  function getUserAccountData(address user) external view returns (UserAccountData memory);
+
+  /// @notice Returns the stored user risk premium.
+  function getUserRiskPremium(address user) external view returns (uint256);
```

Comment:
based on how the other external functions behave in spoke & hub, this function name suggests that this is the up to date user risk premium (from this block). e.g., see `Hub.getAssetDrawnIndex`. Is there a better name for this maybe?